### DISCUSSION TITLE
А как задать новый город, локацию виджета?

### DISCUSSION THREAD

**Discussion by andrew-petrov-lvovitch** [cdek-it/widget#72] [Q&A]
А как динамически задать новый город, локацию виджета? Из данных только название города.

**Comment by vermorag** [comment]
Добрый день. У созданного объекта виджета есть метод [`updateLocation`](https://github.com/cdek-it/widget/blob/main/dist/cdek-widget.es.d.ts#L130C5-L130C19)

**Comment by Andrew100500** [comment]
Вообще defaultLocation толшько не включайте параметр fixBounds: 'locality', иначе получите город в котором нет пунктов выдачи.
